packages/eslint-formatter-gha/src/index.ts

```diff
@@ -75,7 +75,7 @@ const formatter: ESLint.FormatterFunction = (results, data) => {
                 }
             }
             const fileName = `${path.relative(process.cwd(), filePath)}${hash}`;
-            const fileLink = GITHUB_SHA && GITHUB_REPOSITORY ? `https://github.com/${GITHUB_REPOSITORY}/blob/${GITHUB_SHA.slice(0, 7)}/${encodeURI(fileName)}` : "";
+            const fileLink = GITHUB_SHA && GITHUB_REPOSITORY ? `${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/blob/${GITHUB_SHA.slice(0, 7)}/${encodeURI(fileName)}` : "";
```

Comment:
```suggestion
            const fileLink = GITHUB_SHA && GITHUB_REPOSITORY ? `${GITHUB_SERVER_URL ?? "https://github.com"}/${GITHUB_REPOSITORY}/blob/${GITHUB_SHA.slice(0, 7)}/${encodeURI(fileName)}` : "";
```

- :no_entry: Invalid type "string | undefined" of template literal expression.
Rule: [@typescript-eslint/restrict-template-expressions](https://typescript-eslint.io/rules/restrict-template-expressions)
File: [packages/eslint-formatter-gha/src/index.ts#L78](https://github.com/AnnAngela/eslint-packages/blob/2fcb8df/packages/eslint-formatter-gha/src/index.ts#L78)